### PR TITLE
Fix bad 1.8 links

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

File: _posts/2019-08-05-CocoaPods-1.8.0-beta.markdown [hunk 1]
```diff
@@ -11,7 +11,7 @@ _CocoaPods 1.8_ switches the CDN as the default spec repo source and comes with
 
 ### CDN as Default
 
-CDN support was first introduced in the [1.7.2 release](http://127.0.0.1:4000/CocoaPods-1.7.2/), and aims to speed up initial setup and dependency analysis dramatically. With 1.8, CocoaPods _no longer_ requires cloning the now huge [master specs repo](https://github.com/CocoaPods/Specs) in order to function and users may integrate their projects with CocoaPods almost instantly.
+CDN support was first introduced in the [1.7 release](http://blog.cocoapods.org/CocoaPods-1.7.0-beta/) and was finalized in [1.7.2](http://blog.cocoapods.org/CocoaPods-1.7.2/). It aims to speed up initial setup and dependency analysis dramatically. With 1.8, CocoaPods _no longer_ requires cloning the now huge [master specs repo](https://github.com/CocoaPods/Specs) in order to function and users may integrate their projects with CocoaPods almost instantly.
```
Comment (igor-makarov) [inline _posts/2019-08-05-CocoaPods-1.8.0-beta.markdown]: @dnkoutso Safest way to link to other posts:
```
{% post_url 2016-05-04-Master-Spec-Repo-Rate-Limiting-Post-Mortem %}
```
That's the post file name.

Comment (dnkoutso) [reply _posts/2019-08-05-CocoaPods-1.8.0-beta.markdown]: ah nice we can fix that

Comment (igor-makarov) [reply _posts/2019-08-05-CocoaPods-1.8.0-beta.markdown]: I'm on it! BTW the reason it's safer is that if you get the name wrong, Jekyll will throw an error like so:
```
 Liquid Exception: Could not find post "2019-06-14-CocoaPods-1.7.2.markdown" in tag 'post_url'. Make sure the post exists and the name is correct. in _posts/2019-08-05-CocoaPods-1.8.0-beta.markdown
jekyll 2.5.3 | Error:  Could not find post "2019-06-14-CocoaPods-1.7.2.markdown" in tag 'post_url'.
```

Comment (igor-makarov) [reply _posts/2019-08-05-CocoaPods-1.8.0-beta.markdown]: @dnkoutso done in #166 💪

**Pull request merged**